application/controllers/Compte.php

```diff
@@ -0,0 +1,86 @@
+<?php
+defined('BASEPATH') OR exit('No direct script access allowed');
+
+class Compte extends CI_Controller {
+
+	public function index()
+	{
+		$this->load->model('Annonce_model');
+		$data['annonces'] = $this->Annonce_model->get_all_annonces();
+		$this->load->model('Personne_model');
+
+
+		if(isset($this->session->logged_in) && $this->session->logged_in === true)
+		 	{
+		 	$lData['email']=$this->session->email; // prend l'email stocké en session pour aller chercher le reste en base
+		 	$data['utilisateurConnecte'] = $this->Personne_model->get_personne($lData); // recupere en base l'utilisateur grace au mail stocké en session
```

Comment:
A remplacer par une ligne du genre :
$data['utilisateurConnecte'] = $this->Personne_model->get_personne(array('email' => $this->session->email ));